src/components/clusterDetail/ClusterDetailsButtonGroup.css

```diff
@@ -0,0 +1,4 @@
+.vertical-margin {
```

Comment:
We can have name collision here. If you think this is just locally-used class, let's rename it to i.e. `cluster-details-buttons`. If you feel it can be reused, please move it into top-level `styles.css` and prefix the name.